nilearn/regions/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+The :mod:`nilearn.regions` class module includes region extraction
+procedure which operated on a 4D statistical/atlas maps and its functions adapted
+to 3D maps.
+"""
+from .region_extractor import estimate_apply_threshold_to_maps,\
```

Comment:
Yes as an option, it could be also be useful as a seperate functions. If user, has cases like only one 3D map to be extracted. Not only that, we can also fit these functions in code base. But, Where ? I don't know now. Ideas are welcome.